src/routes/AuthLayoutRouter/components/TopNav/TopNav.js

```diff
@@ -105,13 +107,15 @@ function Logo ({ group, isPublic }) {
   )
 }
 
-function Title ({ group, isPublic, onClick }) {
+function Title ({ group, isPublic, onClick, isMyHome }) {
   let [label, name] = ['GLOBAL', 'All My Groups']
 
   if (group) {
     [label, name] = [group.typeDescriptor, group.name]
   } else if (isPublic) {
     [label, name] = ['GLOBAL', 'Public Groups & Posts']
+  } else if (isMyHome) {
+    [label, name] = ['Personal', 'My Home']
```

Comment:
personal should probably be capitalized like GLOBAL is?

Reply:
The styling automatically does this but I'm make it consistent in the copy (since we'll need to add more for i18n changes as well)